src/components/Modal/Modal.tsx

```diff
@@ -121,18 +128,27 @@ export const Modal = ({
     shouldClose.current = false;
   };
 
-  const handleOverlayOnMouseDown = (event) => {
-    if (event.target === modalRef.current) {
+  const handleOverlayOnMouseDown = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (e.target === modalRef.current) {
```

Comment:
We generally try to avoid one character variable names.

Reply:
Changed every `e` to `event` in `Modal` and `ConfirmationModal`. Thanks for the review!